api/types/network/network.go

```diff
@@ -13,6 +13,13 @@ type IPAM struct {
 	Config  []IPAMConfig
 }
 
+// Options represents the network options for endpoint creation
+type Options struct {
```

Comment:
The use of `Options` is confusing in this context. Isnt a better name for this is `EndpointNetworkConfig` ?